word_search.py

```diff
@@ -0,0 +1,102 @@
+'''
+Created on 9 June 2017
+
+@author: Eva
+'''
+from queue import Queue
+
+class WordSearch:
```

Comment:
Yeah I realise that was a really bad practice to use... I will delete that part.

Also I think overall I have confused the order of row and column in my grid (e.g. `grid[row][column]` vs `grid[column][row]`) which is why I probably got very confused, while trying to implement this.